.github/workflows/build.yml

```diff
@@ -52,6 +52,11 @@ jobs:
         file: ./src/Polly.Specs/coverage-reports/Cobertura.xml
         flags: ${{ matrix.os_name }}
 
+    - uses: actions/upload-artifact@v3
+      with:
+        name: mutation-report
```

Comment:
Should we make this name OS-specific so that we know exactly which leg uploaded the report? Or we could only upload it from one OS.

![image](https://user-images.githubusercontent.com/1439341/215533114-7d0363e5-600a-4bdf-9f93-63dfbc1835a6.png)

Reply:
I think it's ok for just picking one, for example (windows + NET7.0). We do the same for our projects. Otherwise, too many reports might overwhelm the users.

Reply:
@martincostello In the end I decided to upload all reports since we are executing the mutation testing on them anyway.